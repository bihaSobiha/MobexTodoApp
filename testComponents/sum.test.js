import 'react-native';
import React from 'react';
import Sum from './sum';
import renderer from 'react-test-renderer';

test('Main SnapShot', () => {
    const snap = renderer.create(
        <Sum />
    ).toJSON();
    expect(snap).toMatchSnapshot();
});