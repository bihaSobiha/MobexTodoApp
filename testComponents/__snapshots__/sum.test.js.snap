// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main SnapShot 1`] = `
<View>
  <Text>
    Sum!
  </Text>
</View>
`;
